Odata-docs/TOC.yml

```diff
@@ -167,6 +167,8 @@
     items:
```

Comment:
Is there a reason the project files are checked in to the docs folders? Can we move them to GitHub?

Reply:
> Is there a reason the project files are checked in to the docs folders? Can we move them to GitHub?

This is GitHub. They need to be in this repo to import as snippets.  For example, the following line
` [!code-csharp[](first-odata-api/samples/2.2/TodoApi/Startup.cs?highlight=6-7&name=snippet_dic)]`

That's how we do all our docs at https://github.com/aspnet/AspNetCore.Docs

See my [ODATA Expand](https://github.com/MicrosoftDocs/OData-docs/pull/74/files) PR you merged.